BGL/examples/BGL_triangulation_2/data/points.xy

```diff
@@ -0,0 +1,5 @@
+0 0
+1 0
+0.2 0.2
+0 1 
+0 2
```

Comment:
There is a permission issue with this file `BGL/examples/BGL_triangulation_2/data/points.xy`. It should not be executable.